django_tutorial/first_project/sba_app/models.py

```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
Oh wait it's because right now we're generating all the front end stuff through the javascript code right?